### PR TITLE
Add oauth access token for Zendesk integration

### PR DESCRIPTION
**Pull request by tasawernawaz**
[LEARNER-2736](https://openedx.atlassian.net/browse/LEARNER-2736)
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [x] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [x] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by tasawernawaz** [comment]
FYI: @edx/learner-spartans

**Review by fredsmith — APPROVED** [review]

**Comment by tasawernawaz** [comment]
@fredsmith Can I merge this ? Is there any need of creating Devops ticket ?

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -950,6 +951,7 @@ edxapp_generic_auth_config:  &edxapp_generic_auth
   YOUTUBE_API_KEY: "{{ EDXAPP_YOUTUBE_API_KEY }}"
   ZENDESK_USER: "{{ EDXAPP_ZENDESK_USER }}"
   ZENDESK_API_KEY: "{{ EDXAPP_ZENDESK_API_KEY }}"
+  EDXAPP_ZENDESK_OAUTH_ACCESS_TOKEN: "{{ EDXAPP_ZENDESK_OAUTH_ACCESS_TOKEN }}"
```
Comment (feanil) [inline playbooks/roles/edxapp/defaults/main.yml:954]: The name in the auth config should not have EDXAPP in it right.  This line should be `ZENDESK_OAUTH_ACCESS_TOKEN: "{{ EDXAPP_ZENDESK_OAUTH_ACCESS_TOKEN }}"`

Can you link to the relevant change in edx-platform?

Comment (tasawernawaz) [reply playbooks/roles/edxapp/defaults/main.yml:954]: @feanil I have created new PR with updated name #4204. https://openedx.atlassian.net/browse/LEARNER-2736 second subtask is where this var will be used.

**Pull request merged**